eland/dataframe.py

```diff
@@ -445,6 +445,16 @@ def drop(
     def __getitem__(self, key):
         return self._getitem(key)
 
+    def __dir__(self):
+        """
+        Provide autocompletion on field names in interactive environment.
+        """
+        return super(NDFrame, self).__dir__() + [
```

Comment:
Is this intentionally `super(NDFrame, self)` instead of `super().__dir__()`?

Reply:
I was writing in p2/3 compatible, but indeed unnecessary here since eland is designed for python3